src/FIO/Action.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Action.h"
+#include "EOS/Name.h" // Name is reused
```

Comment:
this is already included in `Action.h` header

Reply:
true!